### ISSUE TITLE
Document doesn't open on android, no error returned.

### ISSUE THREAD

**Issue by lamuswawir** [syncfusion/flutter-widgets#1061]
This [Document](https://drive.google.com/file/d/1eFbtbP_h-aylXesMT_nh-Z3eCqmTVzby/view?usp=sharing) doesn't open and no error is returned. I don't know what is going on as it appears to load but then just stops. An image is attached to show the result.
![Screenshot_20230201-171921](https://user-images.githubusercontent.com/68683245/216078765-05e6e450-8c67-46bc-afa4-ba5f4baead6e.png)

**Comment by deepika134** [comment]
Hi @lamuswawir , The provided document is corrupted, so the document does not load in the SfPdfViewer. We have checked the document in Adobe Acrobat Pro, which also showed that it was corrupted. Unfortunately, we do not have the support to fix the corrupted entries in the PDF document, as some viewers can do. For your reference, we have attached a screenshot of the issue .
![Picture1](https://user-images.githubusercontent.com/87310214/216596766-673ea9ec-e76a-4885-9bec-660ce6c5642b.png)
